### PR TITLE
LPD-20229 [Marketplace] Create New Solutions base flow

### PR DESCRIPTION
**Pull request by joanagrego**
Related to --> [LPD-20229](https://liferay.atlassian.net/browse/LPD-20229)

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
Just starting reviewing :)

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Solutions/NewSolutionFlow/pages/index.scss [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.solution-company-profile {
+	.ql-container {
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Solutions/NewSolutionFlow/pages/index.scss:2]: what means ql?

**Review by kevenleone — CHANGES_REQUESTED** [review]
Joana looks very good, I left some suggestions for you 🙌🏻

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/PublisherDashboardRouter.tsx [hunk 2]
```diff
@@ -20,6 +20,14 @@ import {AppCreationFlow} from './pages/Apps/AppCreationFlow/AppCreationFlow';
...
 import Projects from './pages/Projects';
 import Solutions from './pages/Solutions';
+import NewSolution from './pages/Solutions/NewSolutionFlow/NewSolutionFlow';
+import SolutionCompanyProfile from './pages/Solutions/NewSolutionFlow/pages/SolutionCompanyProfile';
+import SolutionContactUs from './pages/Solutions/NewSolutionFlow/pages/SolutionContactUs';
+import SolutionCreate from './pages/Solutions/NewSolutionFlow/pages/SolutionCreate';
+import SolutionDetails from './pages/Solutions/NewSolutionFlow/pages/SolutionDetails';
+import SolutionHeader from './pages/Solutions/NewSolutionFlow/pages/SolutionHeader';
+import SolutionProfile from './pages/Solutions/NewSolutionFlow/pages/SolutionProfile';
+import SolutionSubmit from './pages/Solutions/NewSolutionFlow/pages/SolutionSubmit';
```
[the remark this answers is not in the document]
Comment (joanagrego) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/PublisherDashboardRouter.tsx:30]: Done!

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Solutions/NewSolutionFlow/pages/SolutionHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+ */
+
+import {ClayRadio, ClayRadioGroup} from '@clayui/form';
+import {useState} from 'react';
+import ReactQuill from 'react-quill';
+
+import Form from '../../../../../../components/MarketplaceForm';
+import i18n from '../../../../../../i18n';
+
+import 'react-quill/dist/quill.snow.css';
```
[the remark this answers is not in the document]
Comment (joanagrego) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Solutions/NewSolutionFlow/pages/SolutionHeader.tsx:13]: done

**Comment by joanagrego** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Solutions/NewSolutionFlow/pages/index.scss (hunk 1 above)
Comment (joanagrego) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Solutions/NewSolutionFlow/pages/index.scss]: It's the rich text editor css container.

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
Resent --> [PR](https://github.com/is-solutions-delivery/liferay-portal/pull/5635)

**Pull request closed without merging**